.github/workflows/tests.yml

```diff
@@ -0,0 +1,24 @@
+name: Tests
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+  sonarcloud:
```

Comment:
I'm not sure this is meant to say sonarcloud, but isn't that big of a deal.

Reply:
You've spotted my copy and pasting 🙈 sorted 👍